src/fromager/__main__.py

```diff
@@ -249,7 +250,7 @@ def invoke_main() -> None:
         )  # log the full traceback details to the debug log file, if any
         logger.error(f"ERROR: {err}")
         if _DEBUG:
-            raise
+            sys.exit(1)
```

Comment:
This isn't quite what we want.

We always want the exit with an error if there's an exception. When `_DEBUG` is set, then we want to raise the exception again to get the traceback. So, I think we want something like this

```suggestion
            raise
        sys.exit(1)
```

Reply:
Ah okay. Pushed changes again